### DISCUSSION TITLE
Block Facing

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#635] [Help] [answered]
How can I set the facing of a stair?
I think that you can get .blockdata or something, but i dont know where.

**Comment by ghost** [comment, marked as answer]
There should be a "facing" property that you can modify.
Example: ```var block = stairBlock.withProperty("facing", "west");```

I would suggest creating a placement rule for this.

**Comment by ZakShearman** [comment]
Pretty sure it's the Block#withProperties/withProperty and then for stairs the properties should be like "facing" to NORTH, EAST, etc..

You can check out the StairsPlacementRule class for some more information as well as setting the stairs shape.
